Qpf/Macro/Data/Replace.lean

```diff
@@ -146,10 +146,11 @@ def Replace.run : ReplaceM m α → m (α × Replace) :=
 
 -- Have a look at how, e.g., def, deals with binders,
 -- this method might already exist look for BinderView
```

Comment:
```suggestion
/-- Parse `Syntax` for an explicit binder into a `BinderView`.
    This duplicates parts of `Lean.Elab.Term.toBinderViews; this is required because the latter is private -/
```